runtime/common/src/lib.rs

```diff
@@ -68,9 +68,7 @@ impl WeightToFeePolynomial for WeightToFee {
 	type Balance = Balance;
 
 	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
-		// in Rococo, extrinsic base weight (smallest non-zero weight) is mapped to 1 MILLIUNIT:
-		// here, we map to 1/10 of that, or 1/10 MILLIUNIT
-		let p = MILLIUNIT / 10;
+		let p = MILLIUNIT;
```

Comment:
Any doc, why set it to milliunit?

Reply:
According to the formula `GasPrice = NextFeeMultiplier * WeightToFee(WeightPerGas)`, that we use `SlowAdjustingFeeUpdate` as our main `FeeMultiplierUpdate` and I don't want to update it. `WeightPerGas` is a fixed value once the `BLOCKGASLIMIT` and `BLOCKWEIGHTLIMIT` are set. So the only way is to tweak the `WeightToFee ` function to adjust the parameters to get the gas price to fit our needs. The gas price calculated without touching the WeightToFee is `187804800`, which is too low in our case. that why I update this value.